batch/batch/server/server.py

```diff
@@ -631,13 +656,14 @@ def refresh_k8s_state():
 
     log.info('k8s state refresh complete')
 
-    return '', 204
+    return web.Response(status=204)
 
 
-@app.route('/recent', methods=['GET'])
-def recent():
+@routes.get('/recent')
+@aiohttp_jinja2.template('recent.html')
+async def recent(request):  # pylint: disable=W0613
```

Comment:
do we need request here (and the accompanying unused args pylint exception)?

Reply:
It causes an error if I don't have it:
```
ERROR	| 2019-03-11 15:21:27,532 	| web_protocol.py 	| log_exception:355 | Error handling request
Traceback (most recent call last):
  File "//anaconda/envs/hail-batch/lib/python3.6/site-packages/aiohttp/web_protocol.py", line 418, in start
    resp = await task
  File "//anaconda/envs/hail-batch/lib/python3.6/site-packages/aiohttp/web_app.py", line 458, in _handle
    resp = await handler(request)
TypeError: refresh_k8s_state() takes 0 positional arguments but 1 was given
```

Reply:
Ah ok